app/models/manager_refresh/inventory_collection_default/container_manager.rb

```diff
@@ -0,0 +1,326 @@
+        :model_class    => ContainerService,
+        :association    => :container_services,
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_service_port_configs(extra_attributes = {})
+      attributes = {
+        :model_class => ContainerServicePortConfig,
+        :association => :container_service_port_configs,
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_routes(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerRoute,
+        :association    => :container_routes,
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_component_statuses(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerComponentStatus,
+        :association    => :container_component_statuses,
+        :manager_ref    => [:name],
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_templates(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerTemplate,
+        :association    => :container_templates,
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_template_parameters(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerTemplateParameter,
+        :association    => :container_template_parameters,
+        :manager_ref    => [:container_template, :name],
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_builds(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerBuild,
+        :association    => :container_builds,
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def container_build_pods(extra_attributes = {})
+      attributes = {
+        :model_class    => ContainerBuildPod,
+        :association    => :container_build_pods,
+        :manager_ref    => [:namespace, :name],
+        :builder_params => {
+          :ems_id => ->(persister) { persister.manager.id },
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def persistent_volumes(extra_attributes = {})
+      attributes = {
+        :model_class    => PersistentVolume,
+        :association    => :persistent_volumes,
+        :builder_params => {
+          :parent => ->(persister) { persister.manager },
```

Comment:
right, could you do _id and _type then? 
https://github.com/Ladas/manageiq/blob/0d0f45df232ad208efce78aa642ea81d5e0c2b0e/app/models/manager_refresh/inventory_collection_default/cloud_manager.rb#L68-L68

Otherwise the _batch saver could have problems now. I could just extend it though. :-)